### PR TITLE
Iecor11

### PR DESCRIPTION
**Pull request by Bibiko**

### CODE REVIEW

**Review by xrotwang — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -23,31 +23,31 @@ Available online at https://iecor.clld.org
...
 - **Sources:** 0
 - **Synonymy:** 1.01
-- **Cognacy:** 25,928 cognates in 5,013 cognate sets (2,363 singletons)
+- **Cognacy:** 25,741 cognates in 4,981 cognate sets (2,341 singletons)
 - **Cognate Diversity:** 0.19
-- **Invalid lexemes:** 6,762
-- **Tokens:** 90,208
-- **Segments:** 670 (0 BIPA errors, 0 CLTS sound class errors, 670 CLTS modified)
-- **Inventory size (avg):** 37.75
+- **Invalid lexemes:** 6,930
```
[the remark this answers is not in the document]
Comment (Bibiko) [reply README.md:33]: hmm, I have no answer to this question. Maybe the detection was changed due to cldfbench update?

[the remark this answers is not in the document]
Comment (Bibiko) [reply README.md:33]: Is there a way to get the full list of invalid lexemes?

[the remark this answers is not in the document]
Comment (xrotwang) [reply README.md:33]: should be in the transcription report.

[the remark this answers is not in the document]
Comment (Bibiko) [reply README.md:33]: This is really weird. I made a diff of "invalid_words": {...} of version 1.0 and 1.1 and both are exactly the same BUT version 1.0: "invalid_words_count": 6762 and version 1.1: "invalid_words_count": 6930 ????

[the remark this answers is not in the document]
Comment (Bibiko) [reply README.md:33]: I did a diff with "bad_words" and it's fine. In version 1.1 was only one improvement thus version 1.1 has 383 and version 1.0 383.

[the remark this answers is not in the document]
Comment (xrotwang) [reply README.md:33]: Hm. I guess, the cldfbench version (or CLTS version?) used for 1.0 and now differ?

[the remark this answers is not in the document]
Comment (Bibiko) [reply README.md:33]: ah - wait - In `cldf/.transcription-report.json` invalid_words you only find the first 100 invalid lexemes. That list is not the full list.

[the remark this answers is not in the document]
Comment (Bibiko) [reply README.md:33]: I had a quick look at the changes to raw/lexeme.csv, but can't find anything that explains this high number of invalid_lexemes. Mostly it was just changes in comments.

[the remark this answers is not in the document]
Comment (xrotwang) [reply README.md:33]: Hm, I guess the most plausible explanation would be changes in CLTS then.

[the remark this answers is not in the document]
Comment (Bibiko) [reply README.md:33]: > Hm. I guess, the cldfbench version (or CLTS version?) used for 1.0 and now differ?

hmm - nope - CLTS version is the same. I also re-ran makecldf on version 1.0 and it gives: Invalid lexemes: 6762

[the remark this answers is not in the document]
Comment (Bibiko) [reply README.md:33]: > Hm, I guess the most plausible explanation would be changes in CLTS then.

I agree

**Pull request merged**